app/cp_nigeria/views.py

```diff
@@ -1078,6 +1078,8 @@ def cpn_outputs(request, proj_id, step_id=STEP_MAPPING["outputs"], complex=False
     project = get_object_or_404(Project, id=proj_id)
     options = get_object_or_404(Options, project=project)
     report_obj, created = ImplementationPlanContent.objects.get_or_create(simulation=project.scenario.simulation)
+    # saves the graphs and tables to the database if the object doesn't exist yet
+    save_to_db = True if created else False
```

Comment:
As discussed maybe set it to `True` if some fields are missing in the `ImplementationPlanContent` table